R/arx_classifier.R

```diff
@@ -297,10 +327,13 @@ arx_class_args_list <- function(
 
   if (!is.null(forecast_date) && !is.null(target_date)) {
     if (forecast_date + ahead != target_date) {
-      cli::cli_warn(c(
-        "`forecast_date` + `ahead` must equal `target_date`.",
-        i = "{.val {forecast_date}} + {.val {ahead}} != {.val {target_date}}."
-      ))
+      cli_warn(
+        c(
+          "`forecast_date` + `ahead` must equal `target_date`.",
+          i = "{.val {forecast_date}} + {.val {ahead}} != {.val {target_date}}."
+        ),
+        class = "epipredict__arx_args__inconsistent_target_ahead_forecaste_date"
+      )
```

Comment:
Similar comment in the forecaster below.

```suggestion
      cli_warn("`forecast_date`  {.val {forecast_date}} + `ahead` {.val {ahead}} must equal `target_date` {.val {target_date}}.",
        class = "epipredict__arx_args__inconsistent_target_ahead_forecaste_date"
      )
```